### PR TITLE
Add extra devkit instructions to README.md

### PR DESCRIPTION
**Pull request by oct2pus**
included corrections for 2.4+ RubyInstaller based on some issues I had to solve for a newbie who was having issues installing the discordrb gem on post 2.4+ Ruby RubyInstaller.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,11 +77,21 @@ To run the bot:
     ERROR:  Error installing discordrb:
             The 'websocket-driver' native gem requires installed build tools.
 
-You're missing the development kit required to build native extensions. Download the development kit [here](http://rubyinstaller.org/downloads/) (scroll down to "Development Kit", then choose the one for Ruby 2.0 and your system architecture) and extract it somewhere. Open a command prompt in that folder and run:
```
Comment (z64) [inline README.md]: This first line shouldn't be removed, as it no longer reads well with the previous sentence "If you get an error like this..."

Answer that the user is missing the devkit, then go into version specific details.

File: README.md [hunk 2]
```diff
@@ -77,11 +77,21 @@ To run the bot:
...
+**RubyInstaller for ruby 2.4.3-2 and above** RubyInstaller includes the development kit, if you do not have the development kit and have installed ruby using RubyInstaller, open a command prompt and run:
```
Comment (z64) [inline README.md]: This should be a header of an appropriate size, and there is no newline, so it reads awkwardly.

File: README.md [hunk 3]
```diff
@@ -77,11 +77,21 @@ To run the bot:
...
+
+    ridk install
+    
+Select option 3, and then run
+
+    ridk enable
+
+To enable the changes
+
+**RubyInstaller for ruby 2.3.3 and below** You're missing the development kit required to build native extensions. Download the development kit [here](http://rubyinstaller.org/downloads/) (scroll down to "Development Kit", then choose the one for Ruby 2.0 and your system architecture) and extract it somewhere. Open a command prompt in that folder and run:
```
Comment (z64) [inline README.md]: ditto

File: README.md [hunk 4]
```diff
@@ -77,11 +77,21 @@ To run the bot:
...
 
     ruby dk.rb init
     ruby dk.rb install
-
+
```
Comment (z64) [inline README.md]: Remove the trailing whitespace

**Comment by oct2pus** [comment]
Is this better?

**Comment by z64** [comment]
Thanks!

**Pull request merged**